mlflow/utils/credentials.py

```diff
@@ -76,6 +76,27 @@ def login(backend="databricks"):
         )
 
 
+def _is_valid_databricks_host(host):
+    # Check if the host is a valid Databricks URL: 1) start with https://, 2) end with
+    # "databricks.com" or "databricks.com/"
+    if not host.startswith("https://"):
+        _logger.error(f"Host must start with 'https://', but received: {host}")
+        return False
+
+    # Handle the trailing "/" in URL.
+    host = host[:-1] if host.endswith("/") else host
+    if not host.endswith("databricks.com"):
```

Comment:
We can use `urllib.parse.urlparse` and check `params`, `query`, and `fragment` are all empty.

Reply:
to reject a URI like `https://my.databricks.com?foo=databricks.com`.